src/Behat/MinkExtension/ServiceContainer/Driver/ZombieFactory.php

```diff
@@ -44,6 +44,10 @@ public function configure(ArrayNodeDefinition $builder)
                 ->scalarNode('server_path')->defaultNull()->end()
                 ->scalarNode('threshold')->defaultValue(2000000)->end()
                 ->scalarNode('node_modules_path')->defaultValue('')->end()
+                ->arrayNode('options')
+                  ->prototype('scalar')->end()
+                  ->defaultValue(array())
```

Comment:
`->defaultValue(array())` is useless. This is already the case for a prototyped array node